### PR TITLE
[OSCD] Detecting Code injection with PowerShell in another process #70

### PR DESCRIPTION
**Pull request by NikitaStormwind**
https://github.com/Neo23x0/sigma/issues/575

### CODE REVIEW

File: rules/windows/powershell/powershell_code_injection.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+title: Accessing WinAPI in PowerShell. Code Injection.
+id: eeb2e3dc-c1f4-40dd-9bd5-149ee465ad50
+status: experimental
+description: Detecting Code injection with PowerShell in another process
+author: Nikita Nazarov
```
Comment (yugoslavskiy) [inline rules/windows/powershell/powershell_code_injection.yml:5]: the same here (:

File: rules/windows/powershell/powershell_code_injection.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    - attack.t1059.001
+logsource:
+    product: windows
+    service: sysmon
+    definition: 'Note that you have to configure logging for CreateRemoteThread in Symson config'
+detection:
+    selection:
+        EventID:
+            - 8
+        SourceImage: '*\powershell.exe'
```
Comment (yugoslavskiy) [inline rules/windows/powershell/powershell_code_injection.yml:20]: could you please use `endswith ` [value modifier](https://github.com/Neo23x0/sigma/wiki/Rule-Creation-Guide#value-modifiers) here?

File: rules/windows/powershell/powershell_code_injection.yml (hunk 1 above)
Comment (NikitaStormwind) [reply rules/windows/powershell/powershell_code_injection.yml]: Done

File: rules/windows/powershell/powershell_code_injection.yml (hunk 2 above)
Comment (NikitaStormwind) [reply rules/windows/powershell/powershell_code_injection.yml]: Done

**Comment by yugoslavskiy** [comment]
Thank you very much, @NikitaStormwind !

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**